pkg/app/route_wasm_test.go

```diff
@@ -0,0 +1,186 @@
+package app
+
+// import (
```

Comment:
If you are good with the refactor let's remove this file.

Reply:
great teamwork! Thanks again for your help.